cl/api/templates/rest-docs-vlatest.html

```diff
@@ -13,8 +13,29 @@
 {% endblock %}
 
 {% block content %}
-  <div class="col-sm-7">
+  <div class="col-sm-3">
+
+    {% include "includes/toc_sidebar.html" %}
+
+  </div>
+  <div class="col-sm-9">
```

Comment:
Again to mimic other pages:

```suggestion
  <div class="col-xs-12 col-md-8 col-lg-6">
```